analytics_data_api/v0/views/learners.py

```diff
@@ -0,0 +1,308 @@
+"""
+API methods for module level data.
+"""
+from rest_framework import generics, status
+
+from analytics_data_api.constants import (
+    learner
+)
+from analytics_data_api.v0.exceptions import (
+    LearnerEngagementTimelineNotFoundError,
+    LearnerNotFoundError,
+    ParameterValueError,
+)
+from analytics_data_api.v0.models import (
+    ModuleEngagement,
+    ModuleEngagementMetricRanges,
+    RosterEntry
+)
+from analytics_data_api.v0.serializers import (
+    CourseLearnerMetadataSerializer,
+    ElasticsearchDSLSearchSerializer,
+    EngagementDaySerializer,
+    LearnerSerializer,
+)
+from analytics_data_api.v0.views import CourseViewMixin
+from analytics_data_api.v0.views.utils import split_query_argument
+
+
+class LearnerView(CourseViewMixin, generics.RetrieveAPIView):
+    """
+    Get data for a particular learner in a particular course.
+
+    **Example Request**
+
+        GET /api/v0/learners/{username}/?course_id={course_id}
+
+    **Response Values**
+
+        Returns metadata and engagement data for the learner.
+
+            * username: User's username.
+            * enrollment_mode: Enrollment mode (for example, "verified").
```

Comment:
I would remove "enrollment mode" from the description since it's already the field name. Is this the track in which the user is taking a course? Maybe provide another example (audit?).